commands/push.py

```
@@ -0,0 +1,283 @@
+
+
+def find_available_port(start_port=6006):
+    """Find an available port starting from start_port"""
+    import socket
+
+    for port in range(start_port, start_port + 100):
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            try:
+                s.bind(("", port))
+                return port
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)